test/SlowTests/Server/Documents/TimeSeries/TimeSeriesTombstoneCleaner.cs

```diff
@@ -497,6 +506,29 @@ await using (var stream = typeof(TimeSeriesTombstoneCleaner).Assembly.GetManifes
             }
         }
 
+        private async Task<string> GetReplicationDebugInfo(List<RavenServer> nodes, string database)
```

Comment:
note that I think that the evaluation of this message is _before_ and not after the failure

Reply:
no, its after